Software/GuitarPedal/Util/FastSqrt.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <bit>
+#include <cstdint>
+#include <limits>
+
+// https://en.wikipedia.org/wiki/Fast_inverse_square_root
+static constexpr float fastInvSqrt(float number) noexcept
+{
+    //static_assert(std::numeric_limits<float>::is_iec559);
+    //float const y = std::bit_cast<float>(
+    //        0x5F1FFFF9 - (std::bit_cast<std::uint32_t>(x) >> 1));
+    //return y * (0.703952253f * (2.38924456f - (x * y * y)));
+
+
+    // Trouble using bit_cast, not a member of std error even when defining c++20 in makefile, using this found at:
+    //  https://www.geeksforgeeks.org/fast-inverse-square-root/
+    // function to find the inverse square root 
+
+    const float threehalfs = 1.5F; 
+ 
+    float x2 = number * 0.5F; 
+    float y = number; 
+  
+    // evil floating point bit level hacking 
+    long i = * ( long * ) &y; 
+  
+    // value is pre-assumed 
+    i = 0x5f3759df - ( i >> 1 ); 
```

Comment:
[joke/not serious] I see we used the polite version of this instead of the classic version :) You can compare to the version on wikipedia for the true version that I have seen many times 😆